target/bin/addmailuser

```diff
@@ -47,8 +47,11 @@ touch "${DATABASE}"
 (
   flock -e 200
 
-  grep -qi "^$(escape "${USER}")|" "${DATABASE}" 2>/dev/null &&
-    errex "User \"${USER}\" already exists"
+  if grep -qi "^$(escape "${USER}")|" "${DATABASE}" 2>/dev/null
+  then
+    echo "User '${USER}' already exists."
+    exit 0
```

Comment:
`exit 1` should fit better IMO.

Reply:
This will make `setup.sh` print an unexpected error. I found this to be rather misleading, but I can revert back to `exit 1` if you want.

Reply:
If an email address cannot be added, because it's already existing, it's an (unexpected) error. Therefore `exit 0` feels odd to me. But I'll leave it up to you, it's not worth to discuss this one to death 😆

Reply:
Ok. I just saw, you got rid of the MAILARR altogether 👍

Reply:
> Ok. I just saw, you got rid of the MAILARR altogether +1

Belong to the other comment I guess :D

I will revert back to `exit 1` and resolve this conversation:)

Reply:
Yes, I mixed things up. Sorry.